src/main/java/ru/sbtqa/tag/swingback/stepdefs/StepDefinitions.java

```diff
@@ -64,26 +65,28 @@ public Object call() throws Exception {
 
     /**
      * waining in seconds for a specified time
+     *
      * @param wait
-     * @throws InterruptedException
      */
     @And("^wait (.*?) sec$")
     @И("^ожидание (.*?) сек$")
-    public void wait(String wait) throws InterruptedException {
-        SECONDS.sleep(Long.parseLong(wait));
+    public void wait(String wait) {
+        try {
+            SECONDS.sleep(Long.parseLong(wait));
+        } catch (InterruptedException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2142)